src/Components/Text/Text.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react/types-6-0'
+import { Text, TextProps } from './Text'
+
+export default {
+  title: 'Components/Text',
+  component: Text,
+} as Meta
```

Comment:
Might be nice to have it centered in the story :) 
```suggestion
export default {
  title: 'Components/Text',
  component: Text,
  parameters:{
    layout: 'centered'
  }
} as Meta
```